tests/periph_timer_short_relative_set/tests/01-run.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
```

Comment:
201**9** -> typo or because it's a no-changes-copy?

Reply:
fixed